packages/api-client/src/granules.ts

```diff
@@ -163,18 +164,33 @@ export const waitForGranule = async (params: {
 export const reingestGranule = async (params: {
   prefix: string,
   granuleId: GranuleId,
+  collectionId?: CollectionId,
   workflowName?: string | undefined,
   executionArn?: string | undefined,
   callback?: InvokeApiFunction
 }): Promise<ApiGatewayLambdaHttpProxyResponse> => {
-  const { prefix, granuleId, workflowName, executionArn, callback = invokeApi } = params;
+  const {
+    prefix,
+    granuleId,
+    collectionId,
+    workflowName,
+    executionArn,
+    callback = invokeApi,
+  } = params;
+
+  let path = `/granules/${collectionId}/${granuleId}`;
+
+  // Fetching a granule without a collectionId is supported but deprecated
+  if (!collectionId) {
+    path = `/granules/${granuleId}`;
+  }
```

Comment:
All of this logic will be removed in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3015. We will default to the path including `collectionId`